pytorch_lightning/loggers/tensorboard.py

```diff
@@ -47,6 +49,26 @@ def __init__(self, save_dir, name="default", version=None, **kwargs):
         self.tags = {}
         self.kwargs = kwargs
 
+    @property
+    def root_dir(self):
+        """ Parent directory for all tensorboard checkpoint subdirectories.
+            If the experiment name parameter is None or the empty string, no experiment subdirectory is used
```

Comment:
there is no for extra tab spacing

Reply:
I think it now follows the format used elsewhere.  We're getting there.

Reply:
the point/issue is that still do not have unified formatting everywhere yet...